src/neo-vm/Types/String.cs

```diff
@@ -6,46 +6,46 @@
 namespace Neo.VM.Types
 {
     [DebuggerDisplay("Type={GetType().Name}, Value={System.BitConverter.ToString(Memory.ToArray()).Replace(\"-\", string.Empty)}")]
-    public class ByteArray : PrimitiveType
+    public class String : PrimitiveType
```

Comment:
Maybe we should avoid names defined in `System` namespace, it could be confused for developers. Maybe we should add `Item` as a suffix

Reply:
`ArrayItem` can be misleading as "item of array".

Reply:
VMArray, VMString?

Reply:
It's already in `NeoVM.Types.String`, I think we don't need `VM` prefix again?